src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1066,8 +1066,6 @@ void setOptimalIndex(int newOptimalIdx)
          */
         void rtcpTransform(ByteArrayBuffer baf)
         {
-            SeenFrame maxSentFrame = this.maxSentFrame;
-
```

Comment:
@gpolitis after looking even further at this, i'm still a bit confused.  this assignment doesn't actually protect this method from `maxSeenFrame` changing behind its back (as we just create an alias here, not a deep copy of the member), so this will only protect us from the other thread setting `this.maxSeenFrame` to null.  it won't, for example, prevent the other thread from changing `maxSeenFrame.tsTranslation` to null behind our back (which could theoretically cause an NPE here)